source/reference/minio-mc/mc-alias-set.rst

```diff
@@ -198,4 +198,14 @@ For more complete documentation on S3 Access Control, see
 :s3-docs:`Amazon S3 Security <security.html>`.
 
 For all other S3-compatible services, defer to the documentation for that
-service.
\ No newline at end of file
+service.
+
+Server-Side Encryption
+~~~~~~~~~~~~~~~~~~~~~~
+
+For deployments with :ref:`server side encryption <minio-sse>` enabled, the MinIO Client asks the user whether to trust the certificates for the deployment when creating the alias.
+
+If trusted, the MinIO Client automatically adds the certificate and certificate authority to:
+
+-  ``~/.mc/certs/`` on Linux and other Unix-like systems.
+-  ``C:\Users\[username]\mc\certs`` on Windows systems.
```

Comment:
Ah - this is the SSE-specific bit. So this only works for when we have SSE enabled?

What CA are we pulling down here? I assume the MinIO Deployment CA, not the KES or External KMS CA?

I'm a bit surprised we can download the MinIO CA at all - that sounds like new behavior?

Reply:
Cleaned this up, as I was not understanding correctly.